ansible/roles/node/tasks/main.yml

```diff
@@ -59,7 +59,7 @@
     - restart kubelet
 
 - name: Enable kubelet
-  service: name=kubelet enabled=yes 
+  service: name=kubelet enabled=yes state=started 
```

Comment:
This should happen in the `restart kubelet` handler I thought to prevent numerous wasted bounces